corpora/shakespeare_plus/scripts/process_corpus.py

```
@@ -0,0 +1,342 @@
+        print(f"{'='*40}")
+
+        all_samples = []
+
+        for filepath in sorted(files):
+            samples = process_file(filepath)
+            if samples:
+                print(f"  {filepath.stem}: {len(samples)} chunks")
+                all_samples.extend(samples)
+
+        if not all_samples:
+            print(f"  No samples generated!")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)